app/src/repositories/webtoon/detailRepository.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+const db = require("../../config/db");
+
+class DetailRepository {
+  // 웹툰 상세 정보 불러오기
+  async getWebtoonByWebtoonId(webtoonId) {
+    const query = `
+      SELECT 
+        id, 
+        title, 
+        writer, 
+        illustrator, 
+        day_of_week, 
+        age_rating, 
+        description, 
+        thumbnail_url, 
+        favorite_count
+      FROM webtoons
+      WHERE id = ?
```

Comment:
다른 PR에서 `;`이 대부분 붙이게끔 변경된거로 봤었는데 붙이는거로 논의된게 맞을까요??

```suggestion
      WHERE id = ?;
```

Reply:
네 맞습니다! `;` 붙이는 걸로 정리되었고, 앞으로도 쿼리 작성 시 신경 쓰겠습니다. 확인 감사합니다!